pom.xml

```diff
@@ -8,5 +8,17 @@
     <artifactId>threadConcurrency</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>9</source>
+                    <target>9</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
```

Comment:
empty line in the end of file (btw it's always better to add travis to your projects to catch some codestyle error)